frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -522,7 +522,7 @@ export function FieldValuesWidgetInner({
     function (option: FieldValue): {
       label: string;
       value: string;
-      customLabel?: string;
+      customlabel?: string;
```

Comment:
Looks like this is not referenced anywhere else is this PR - is this just a name change to make it consistent with `label`?